packages/camera/camera/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
-## NEXT
+## 0.10.0
 
+* **Breaking Change** Bumps default camera_web package version to 0.3.0 and camaera_android package version to 0.10.0,
```

Comment:
Optional nit: I wouldn't mention the implementation package versions since that's largely an implementation detail of federation; I would just say the part about the exception code changes since that's the actual breaking change.